src/software/geom/polygon.h

```diff
@@ -55,6 +55,16 @@ class Polygon : public virtual Shape
      */
     const std::vector<Point>& getPoints() const;
 
+    /**
+     * Creates a polygon that is oriented along the vector between the first and second
+     * point. The polygon contains both points as "focal" points similar to an ellipse.
+     * see definition for details
+     * @param firstPoint the first point
+     * @param secondPoint the second point.
+     * @return an oriented polygon along the 2 points that contains both points.
```

Comment:
javadocs seem out of date, it's also unclear from the javadocs if the radius is from segment end to the nearest vertices of the polygon or to the nearest edge. I think a few words would help clear that up